internal/integration/cockroach_test.go

```diff
@@ -78,12 +78,6 @@ func crdbRun(t *testing.T, fn func(*crdbTest)) {
 	}
 }
 
-func TestCockroach_EntRevisions(t *testing.T) {
-	crdbRun(t, func(t *crdbTest) {
-		testEntRevisions(t)
-	})
-}
-
```

Comment:
Very crucial test. I don't feel well to remove them from master, even for a small time-frame.

Reply:
We talked about it.